internal/core-macros/lib.rs

```diff
@@ -44,6 +48,22 @@ pub fn slint_element(input: TokenStream) -> TokenStream {
                 pub_prop_field_names.push(name);
                 pub_prop_field_types.push(&field.ty);
             }
+
+            if let syn::Type::Path(syn::TypePath { path: syn::Path { segments, .. }, .. }) =
+                &property_type
+            {
+                if let Some(syn::PathSegment { ident, arguments: syn::PathArguments::None }) =
+                    segments.first()
+                {
+                    if *ident == "Coord" {
```

Comment:
instead of special casing for Property<Coord> and add the logical_,  couldn't we just change the `Item`s to use `Property<LogicalLength>` ?  i'd prefer that, but of course this is more intrusive.

Reply:
Happy to give this a try. The thought had also crossed my mind. I'd create a separate PR though.